### PR TITLE
Meat grinding for synths fix

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

image

Added /meat to chemistry gripper list and allowed synthetics to interact with the grinder.

## Why It's Good For The Game

Fixes cool.

## Changelog
:cl:
fix: chemistry gripper not accepting meat
/:cl:

### CODE REVIEW

File: code/modules/reagents/machinery/grinder.dm [hunk 1]
```diff
@@ -76,9 +76,6 @@
 	SSnano.update_uis(src)
 	return 0
 
-/obj/machinery/reagentgrinder/attack_ai(mob/user as mob)
```
Comment (vode-code) [inline code/modules/reagents/machinery/grinder.dm:79]: does the usability tweak in attackby not work? attack_ai, as the proc name implies, is for AIs, not direct touching.
`//Useability tweak for borgs`
`if (istype(I,/obj/item/gripper))`
`	ui_interact(user)`
`	return`

Comment (SirRichardFrancis) [reply code/modules/reagents/machinery/grinder.dm:79]: Gripper transfer items just fine, but grinder's interface can't be opened by silicon.

Comment (vode-code) [reply code/modules/reagents/machinery/grinder.dm:79]: and clicking with an empty gripper doesn't work?

Comment (SirRichardFrancis) [reply code/modules/reagents/machinery/grinder.dm:79]: It doesn't.

Comment (SirRichardFrancis) [reply code/modules/reagents/machinery/grinder.dm:79]: Regardless, I see no reason why robot can remotely interface with almost any machine, but not an industrial grinder.
Some modules have to interact with it as part of their "job".

Comment (vode-code) [reply code/modules/reagents/machinery/grinder.dm:79]: I thought maybe it'd let them teleport beakers, but from what I saw the beakers and inserted items eject at the location of the grinder. Not sure if there's a button on the UI that lets you teleport stuff in though.

Comment (SirRichardFrancis) [reply code/modules/reagents/machinery/grinder.dm:79]: If such problem even exist, there is a better way of fixing it than simply locking machine for robots.

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**